data/crac-file/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/remedial_action/range_action/PstRange.java

```diff
@@ -130,10 +154,4 @@ private PhaseTapChanger checkValidPstAndGetPhaseTapChanger(Network network) {
         }
         return phaseTapChanger;
     }
-
-    public int getCurrentTapPosition(Network network) {
-        TwoWindingsTransformer transformer = network.getTwoWindingsTransformer(networkElement.getId());
-        PhaseTapChanger phaseTapChanger = transformer.getPhaseTapChanger();
-        return phaseTapChanger.getTapPosition();
```

Comment:
Wasn't it a nice method? :cry:

Reply:
I think it is better to access the network once in the synchronize method, and to save the value of the current tap position in an attribute. Moreover it makes it possible to remove an argument from some methods signatures.